src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/SmoModel/SmoTreeNodes.cs

```diff
@@ -921,6 +921,20 @@ public override IEnumerable<INodeFilter> Filters
             }
         }
 
+        public override IEnumerable<NodeSmoProperty> SmoProperties
+        {
+            get
+            {
+                var properties = new List<NodeSmoProperty>();
+                properties.Add(new NodeSmoProperty
+                {
+                   Name = "LedgerViewType",
```

Comment:
>"LedgerViewType" [](http://example.com/codeflow?start=26&length=16)

nameof(View.LedgerViewType)

using nameof 
1. Eliminates typos
2. Enables easier search in IDEs with "Find All References"

Reply:
not possible here.. codegen..